airbyte-integrations/connectors/source-cockroachdb/src/main/java/io/airbyte/integrations/source/cockroachdb/CockroachDbSource.java

```diff
@@ -90,6 +97,32 @@ public JsonNode toDatabaseConfig(final JsonNode config) {
     return super.read(config, catalog, state);
   }
 
+  @Override
+  public Set<JdbcPrivilegeDto> getPrivilegesTableForCurrentUser(final JdbcDatabase database, final String schema) throws SQLException {
```

Comment:
Could you please add an integration test to check this logic:
create a table with data for user and then revoke privileges to see if the table then appear in the list for example.
Tnx

Reply:
Added. Thanks.